rfcs/0027-formalize-documentation-organization.md

```diff
@@ -0,0 +1,430 @@
+to make it easier for entire TVM community, to easily find documentation to
+meet their needs, whether they are new users or experienced users. Another goal
+is to make it easier for the development community to contribute to the TVM
+documentation. It recognizes that while in most communities there is a distinct
+divide between the user and the developer communities, there can be significant
+overlap given the nature of TVM as an optimizing compiler.
+
+# Motivation
+[motivation]: #motivation
+
+TVM has seen an explosion of growth since it was released as an open source project,
+and formally grauated as an official Apache Software Foundation project. The vision of
+the Apache TVM Project is to host a "diverse community of experts and
+practitioners in machine learning, compilers, and systems architecture to build
+an accessible, extensible, and automated open-source framework that optimizes
+current and emerging machine learning models for any hardware platform."
+
+The TVM community has done an excellent job in producing a wide range of documents to describe
+how to successfully install, use, and develop for TVM. The documenation project grew with
+the community, to address the immediate needs of the developer community. However, one
+consistent piece of feedback is that the documentation is difficult to navigate, with beginner
+material mixed in with advanced material. As a result, it can be difficult for new users
+to find the exact information they need, and can work against the vision of the project.
+
+This RFC aims to refactor the organization of the TVM docs, loosely following the [formal
+documentation style described by Divio](https://documentation.divio.com). This system has been chosen
+because it is a:
+
+> "simple, comprehensive and nearly universally-applicable scheme. It is proven
+> in practice across a wide variety of fields and applications."
+
+This RFC is primarily concerned with the organization of the documents, and not
+the content. As such, the implementation of this RFC would move documents, and
+only create new documents as top-level placeholders, indexes, and documentation
+about the system itself.
+
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## The Four Document Types
+
+### Introductory Tutorials
+
+These are step by step guides to introduce new users to a project. A successful
+introductory tutorial successfully get the user engaged with the software
+without necessarily explaining why the software works the way it does. Those
+explanations can be saved for other document types; the introductory tutorial
+focuses on a successful first experience. These are the most important docs to
+turning newcomers into new users and developers. A fully end-to-end tutorial,
+from installing TVM and supporting ML software, to creating and training a
+model, to compiling to different architectures will give a new user the
+opportunity to use TVM in the most efficient way possible.
```

Comment:
could you explain more the difference between a tutorial and a how-to? Right now, "A fully end-to-end tutorial,
from installing TVM and supporting ML software, to creating and training a model, to compiling to different architectures" would likely have some step-by-step component. Can you explain what additional content should be in here to make it a tutorial?

Reply:
@areusch I've updated a bit, does this look better now?